### PR TITLE
[BACK-2744] Add AnyHigh and AnyLow to Summaries

### PR DESCRIPTION
**Pull request by Roukoswarf**

### CODE REVIEW

**Review by ewollesen — DISMISSED** [review]
LGTM

As far as I can tell, this is just updates to use newer libraries, and newer versions of generated specs.

File: spec/summary.v1.yaml [hunk 1]
```diff
@@ -80,14 +80,12 @@ components:
         - bgm
       description: Field which contains a summary type string.
       x-go-type: string
+      x-go-type-skip-optional-pointer: true
     BGMBucketData:
       type: object
       title: bgmbucketdata.v1
       description: Series of counters which represent one hour of a users data
       properties:
-        date:
```
Comment (ewollesen) [inline spec/summary.v1.yaml:88]: Is this intentional?

**Review by ewollesen — DISMISSED** [review]
Just one small question, but assuming it's intention, then it looks fine.

File: spec/summary.v1.yaml (hunk 1 above)
Comment (Roukoswarf) [reply spec/summary.v1.yaml:88]: yes, vestigial error correction.

**Pull request merged**